cmd/tink-worker/cmd/root.go

```diff
@@ -3,16 +3,18 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"os"
 	"strings"
 	"time"
 
+	dockerCli "github.com/docker/docker/client"
```

Comment:
Relatively minor: alias's should be all lowercase same as package names.

Reply:
Fixed